docker-compose.yml

```
@@ -1,106 +1,231 @@
+      - MINIO_PASSWORD=miniopassword
+      - MINIO_BUCKET=scraped
+    depends_on:
+      - neo4j
+      - minio
+    restart: unless-stopped
+
+  neo4j:
+    image: neo4j:latest
+    container_name: neo4j
+    ports:
+      - "7474:7474" # Browser
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)